.github/ISSUE_TEMPLATE.md

```diff
@@ -16,3 +16,9 @@ EXPLAIN WHAT IS TO BE EXPECTED, HERE.
 
 ### Actual Behavior
 EXPLAIN WHAT IS ACTUALLY HAPPENING, HERE.
+
+P.S. Remember, an issue is not the place to ask questions. You can use [Stack Overflow](http://stackoverflow.com/questions/tagged/cakephp)
+for that or join the #cakephp channel on irc.freenode.net, where we will be more
+than happy to help answer your questions.
+
+Before you open an issue, please check if a similar issue already exists or has been closed before.
```

Comment:
@ionas I too would prefer having it at the top RIGHT IN THE FACE, but others might not share the same opinion.

@markstory Probably, but IMO having this disclaimer is necessary and hence I have kept it at bottom.